tests/Feature/TrafficAnalytics/ApplicationAnalyticsTest.php

```
@@ -0,0 +1,216 @@
+        '/traffic/attribution' => json_encode(['attribution' => 'GeoIP data by MaxMind']),
+    ];
+}
+
+beforeEach(function () {
+    $this->team = Team::factory()->create();
+    $this->user = User::factory()->create();
+    $this->team->members()->attach($this->user->id, ['role' => 'owner']);
+    $this->actingAs($this->user);
+    session(['currentTeam' => $this->team]);
+
+    $this->privateKey = PrivateKey::create([
```

PHPStan (2.2.7)

[warning] Call to an undefined static method App\Models\PrivateKey::create().


(staticMethod.notFound)